generate_ics.py

```diff
@@ -126,6 +127,7 @@ def main():
                         subject_code)
 
                 subjects[subject_code] = str(summary)
+                update_sub_list(subject_code, summary)
```

Comment:
What is the point of updating on the user's file? The user will have to commit it and push it to GitHub / open a PR for it to reflect on the main repository.

Reply:
Well, for beginning the user won't have to enter the information again while buildig ICS. Also, a message has been added to request the user to send a PR.

Reply:
Yeah, adding a message is good.

Remember that once a user generates the ICS file, they will most probably come back the next semester. Since very few people will take the same subject twice, adding locally wouldn't add much value. 🙂 If they did open a PR though, it would be very useful.